test/testText.cpp

```diff
@@ -129,10 +129,10 @@ TEST_CASE("Text Basic", "[tvgText]")
     REQUIRE(Text::load(TEST_DIR"/Arial.ttf") == tvg::Result::Success);
     REQUIRE(text->font("Arial", 80) == tvg::Result::Success);
 
-    REQUIRE(text->text("THORVG Text") == tvg::Result::Success);
-    REQUIRE(text->text("ABCDEFGHIJIKLMOPQRSTUVWXYZ") == tvg::Result::Success);
-    REQUIRE(text->text("") == tvg::Result::Success);
     REQUIRE(text->text(nullptr) == tvg::Result::Success);
+    REQUIRE(text->text("") == tvg::Result::Success);
+    REQUIRE(text->text("ABCDEFGHIJIKLMOPQRSTUVWXYZ") == tvg::Result::Success);
+    REQUIRE(text->text("THORVG Text") == tvg::Result::Success);
```

Comment:
Wondering how this changes improve test coverage. I see the code lines are the same.

Reply:
from tvgText.h text(): 
```
        free(this->utf8);
        if (utf8) this->utf8 = strdup(utf8);
```
text is overwritten , so if the last one passed was 'nullptr' not much was really tested in the ttf loader